pl/firefox/australis/firefox_tour.lang

```diff
@@ -26,7 +26,7 @@ Dodawaj i przeglądaj zakładki w jednym miejscu
 
 
 ;Learn more about the new Firefox
-Dowiedz się więcej o nowym Firefoksie
+Więcej informacji o nowym Firefoksie
```

Comment:
I'm not sure if something broke with GitHub recently, I've seen it in another repository.
https://github.com/piotrdrag/www.mozilla.org/blob/ea0bdee4f0978e755db3edb9b8f2ae3a20ff34f5/pl/firefox/australis/firefox_tour.lang#L28